.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,201 @@
+        If you are encountering the issue with multiple different configurations, please list them.
+        This helps us to be able to reproduce the issue, or help us to investigate why this issue occurs
+        for some users and not others.
+
+  - type: dropdown
+    id: nvdaType
+    attributes:
+      label: NVDA type
+      description: Is this reproduced with NVDA installed/portable/running from source?
+      multiple: true
+      options: ["installed copy", "portable copy", installer/launcher copy", "source copy"]
+    validations:
+      required: true
+
+  - type: input
+    id: nvdaVersion
+    attributes:
+      label: NVDA version
+      description: |
+        Which version(s) of NVDA can you reproduce this issue with?
+        This can be copied from the about dialog, from the NVDA help menu.
+    validations:
+      required: true
+
+  - type: textarea
+    id: otherNvdaVersions
+    attributes:
+      label: Have you tried any other versions of NVDA? If so, please report their behaviors.
+      description: |
+        Knowing whether this issue occurs in previous releases helps us to understand if this is a newly introduced issue.
+        It can help us determine if changes in NVDA or other software introduced the problem.
+
+        *Example:*
+        > * NVDA 2018.1 - no error sound
+        > * NVDA 2018.2 - has error sound
+
+  - type: input
+    id: winVersion
+    attributes:
+      label: Windows version
+      description: |
+        The version of Windows that you are using.
+        You can copy this from the "System Information" application:
+        1. Press `Windows+r` to open the run dialog
+        1. Type `msinfo32` and press enter
+        1. From the "System summary" tree-view item, press tab to get to the right pane.
+        1. Press down arrow twice to get to "Version"
+        1. Press `Control+C` to copy the version information.
+
+        *Example:*
+        > Windows 10 Version 1607 Build 14393.1066
+    validations:
+      required: true
+
+  - type: input
+    id: otherVersions
+    attributes:
+      label: Name and version of other software in use when reproducing the issue
+      description: |
+        Typically, you can get this information from the "About" dialog of the software.
+
+        *Example:*
+        > Chrome version: 67.0.3396.87
+
+  - type: textarea
+    id: otherSysInfo
+    attributes:
+      label: Other information about your system
+      description: |
+        This is other information about your setup that you think might be relevant to us.
+
+        *Example:*
+        > Running Windows 10 in a VM
+
+  - type: markdown
+    attributes:
+      value: |
+        ### Troubleshooting steps
+        The following questions ask about the basic investigation steps you may have completed.
+        Please feel free to add more information here to tell us about what you have tried.
+
+  - type: input
+    id: restart
```

Comment:
Could this become a checkbox?

Reply:
It seems that the checkbox UX is more designed for what we do with pull requests, a list of checkboxes to confirm with.
I think using `input` allows people to add additional notes if relevant as well.